manage-accounts-specs/src/test/groovy/uk/gov/justice/digital/hmpps/manageusers/mockapis/OauthApi.groovy

```diff
@@ -71,6 +71,7 @@ class OauthApi extends WireMockRule {
         this.stubFor(
                 get(urlPathEqualTo("/auth/api/user/me/roles"))
                         .willReturn(aResponse()
+                        .withHeader('Content-Type', 'application/json')
```

Comment:
this was causing superagent to treat the response as `text` instead of `json` so wasn't parsing the body